.github/workflows/integrate-landingpages-frontend.yml

```
@@ -2,10 +2,20 @@ name: "Integrate"
 
 on:
   workflow_call:
+    secrets:
+      GH_PACKAGES_INSTALL_TOKEN:
+        required: false
+        description: 'GitHub Token used to install private NPM packages from GitHub Packages registry.'
+    inputs:
+      installsPrivatePackage:
+        description: 'Determine if private NPM packages from Github Packages registry should be installed.'
+        required: false
+        type: string
+        default: false
 
 jobs:
   frontend-build:
-    if: github.actor == 'dependabot[bot]'
+    # if: github.actor == 'dependabot[bot]'
```

Revert these changes, before this PR is merged.

```suggestion
    if: github.actor == 'dependabot[bot]'
```